nailgun/entities.py

```diff
@@ -1670,12 +1670,21 @@ def create_missing(self):
                 'Found instance attributes: {0}'.format(self.get_values())
             )
         super(Host, self).create_missing()
+        self.name = self.name.lower()
```

Comment:
Host name is created in lowercase. create_missing() is generating name with both lower- and uppercase symbols. If we don't lower it here, than we'll receive ```self.name``` e.g. ```TeStHoSt```, when the actual name of created host is ```testhost```. It causes problem with, for example, search (```hosts.search()``` func). So we either generate name here properly or have to lower received name each time we use the host in our tests.